src/pages/donation.tsx

```diff
@@ -26,6 +26,22 @@ export default function DonationPage() {
             現在のところ活動資金は寄付によって賄われていますが、十分とはいえない状況です。
             皆様のご支援をいただけますと幸いです。
           </p>
+          <h2>協賛による支援</h2>
+          <p>
+            {/* eslint-disable prettier/prettier */}
+            企業様からの資金協賛による支援も募集しています。
```

Comment:
[NITS]
```suggestion
            企業様からの資金協賛による支援を募集しています。
```
「も」=>「を」かなと思った。